modules/persistent-storage-csi-gcp-pd-encrypted-pv.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-gcp-pd.adoc
+
+[id="persistent-storage-csi-gcp-pd-encrypted-pv_{context}"]
+= Creating a custom-encrypted persistent volume
+
+When you create a `PersistentVolumeClaim` object, {product-title} provisions a new persistent volume (PV) and creates a `PersistentVolume` object. You can add a custom encryption key in Google Cloud Platform (GCP) to protect a PV in your cluster by encrypting the newly created PV.
+
+For encryption, the newly attached PV that you create uses customer-managed encryption keys (CMEK) on a cluster by using a new or existing Cloud KMS key.
+
+.Prerequisites
+* You are logged in to a running {product-title} cluster.
+* You have created a Google Cloud Key Management Service (KMS) key ring and key version.
+
+For more information about CMEK and Cloud KMS resources, see link:https://cloud.google.com/kubernetes-engine/docs/how-to/using-cmek[Using customer-managed encryption keys (CMEK)].
+
+.Procedure
+To create a custom-encrypted PV, complete the following steps:
+
+. Create a storage class with the Cloud KMS key. The following example enables dynamic provisioning of encrypted volumes:
++
+[source,yaml]
+--
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: csi-gce-pd-cmek
+provisioner: pd.csi.storage.k8s.io
```

Comment:
Provisioner name should update. Pay attention to this [bug](https://bugzilla.redhat.com/show_bug.cgi?id=1915608
)

Reply:
OK, wondered about that...I've updated to follow @jsafrane's recommendation in the BZ for using `pd.csi.storage.gke.io`.